topologies/datacenter/configlets/Leaf1-VXLAN-Lab

```diff
@@ -63,18 +63,18 @@ interface Vxlan1
    vxlan flood vtep 172.16.0.56
    vxlan vlan 12 vni 1212
 !
-ip virtual-router mac-address 00:1c:73:00:00:12
```

Comment:
@bingbucks what is the purpose of changing the virtual-router mac-address?  Looks like the last set corresponds to leaf pairs.  Any particular reason moving it to `ff`?

Reply:
Good question. The reason I did that was so that an ATD user didn't get confused about the fact that the virtual MAC is the same on each side of the VxLAN tunnel. In this particular lab you are referring to the vMACs that both ended in ":12" which is fine. But I wanted it to be more apparent that ":12" was NOT anything special and had nothing to do with Leaf1 & Leaf2 given our naming conventions (the other labs have vMACs with ":12" to denote the Leaf1 & Leaf2 MLAG pair and ":34" to denote the Leaf3 & Leaf4 MLAG pair). That ":12" and ":34" naming convention communicated to the user that each vMAC is unique do the leaf pairs. But in the L2eVPN lab, the vMACs are the same everywhere and I wanted to make it more apparent that ":FF" is NOT tied to any particular side of the fabric.

Does that make sense? Do you think it would be more clear to an ATD user by changing it to ":FF"?

Reply:
Thanks for the clarification and I can follow up.